en/includes/references/conditional-auth/api-reference.md

```diff
@@ -904,13 +907,15 @@ The HTTP GET function enables sending HTTP GET requests to specified endpoints a
             "Accept": "application/json"
         }, authConfig, {
             onSuccess: function(context, data) {
-                Log.info('httpGet call succeeded');
-                context.selectedAcr = data.status;
```

Comment:
why we remove these lines. IMO, this shows how data and context is used. Thus it's a meaningful example

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4773/commits/9acc22f1d66873e2c67cb71939862187674bece9